app/mailers/resend_link_request_mailer.rb

```diff
@@ -2,14 +2,14 @@ class ResendLinkRequestMailer < GovukNotifyRails::Mailer
   require_relative 'concerns/notify_template_methods'
   include NotifyTemplateMethods
 
-  def notify(legal_aid_application)
+  def notify(legal_aid_application, applicant, to = support_email_address)
     template_name :new_link_request
     set_personalisation(
-      applicant_name: legal_aid_application.applicant_full_name,
-      provider: legal_aid_application.provider_id,
-      application_ref: legal_aid_application.application_ref,
+      applicant_name: "#{applicant['first_name']} #{applicant['last_name']}".strip,
```

Comment:
This is in case the first_name or the last_name is missing.
" doe" -> "doe"
"john " -> "john"
But we have validation on these attributes so it should not happen.
I'll change the code to your suggestion

Reply:
Ah. I see how it's working. If `applicant['first_name']` is blank, the test is `" Jones"` and the `strip` tidies it up to just `"Jones"`. Sorry, my miss reading of the code. I'd normally handle that with:
```ruby
[applicant['first_name'], applicant['last_name']].compact.join(' ')
```
But the outer `strip` method is simpler. Leave this as it is.